spec/src/main/asciidoc/circuitbreaker.asciidoc

```diff
@@ -59,26 +53,46 @@ public Connection serviceA() {
    return conn;
 }
 ----
-Below are a couple scenarios to demonstrate when the circuit opens for the above code snippet.
 
 * Scenario 1
 ** Request 1 - success
 ** Request 2 - failure
-** Request 3 - failure
+** Request 3 - success
 ** Request 4 - success
-** Request 5 - CircuitBreakerOpenException
+** Request 5 - failure
+** Request 6 - CircuitBreakerOpenException 
 
-The Request 4 will trigger the circuit to open. The Request 5 will hit the exception.
+In this scenario, request 5 will trigger the circuit to open because out of the last four requests (the `requestVolumeThreshold`), two failed
+which reaches the `failureRatio` of `0.5`. Request 6 will therefore hit the `CircuitBreakerOpenException`.
 
 * Scenario 2
 ** Request 1 - success
 ** Request 2 - failure
-** Request 3 - success
+** Request 3 - failure
 ** Request 4 - success
-** Request 5 - failure
-** Request 6 - CircuitBreakerOpenException 
+** Request 5 - CircuitBreakerOpenException
+
+In this scenario, request 4 will cause the circuit to open. Request 5 will hit the `CircuitBreakerOpenException`.
```

Comment:
It seems you just swapped the 2 scenarios, do I get that correctly?

Reply:
Yeah, I did (didn't realise the diff would look so odd). I added some further explanation to what was happening in each scenario and I realised that scenario 2 was the "normal" one where the rolling window is full when the failure ratio is reached and scenario 1 was the less usual one where the failure ratio is reached before the rolling window is full but the circuit doesn't open until the window is filled. I swapped them to put the more common and easier to understand scenario first.

Reply:
Agree, it just took me a while to understand the diff, so I figured I'd rather ask :-)